docs/learn/digital-assets/read-asset-data.md

```diff
@@ -94,95 +107,119 @@ const isLSP8 = await myAsset.supportsInterface(
+| `LSP4CreatorsMap:<address>`           | A mapping of creator-specific information.                                                                                                                                                  |
+| `LSP4TokenType`                       | The asset's token type (encoded as a `uint256` number).                                                                                                                                     |
+| `LSP4TokenSymbol`                     | The utf8 hex characters of the string representing the asset's symbol.                                                                                                                      |
+| `LSP4TokenName`                       | The utf8 hex characters of the string representing the asset's name.                                                                                                                        |
 
-If both, contract and metadata standard are aligned, we can continue to determine the [LSP4 Token Type](../../standards/tokens/LSP4-Digital-Asset-Metadata.md#types-of-digital-assets) in order to interpret the metadata correctly.
+To retrieve and decode the value stored under this data keys, we will use [erc725.js](../../tools/erc725js/getting-started.md) and instantiate the class with:
 
-<!-- prettier-ignore-start -->
+1. the LSP4 Schema that defines the data keys above and how to decode them.
+2. the address of your digital asset.
+3. an RPC URL.
+4. an IPFS gateway to fetch any off-chain data linked to the contract storage.
 
-```js
-// ...
+```ts title="Instantiate erc725.js"
+import { ERC725 } from '@erc725/erc725.js';
+import lsp4Schema from '@erc725/erc725.js/schemas/LSP4DigitalAsset.json';
+import { INTERFACE_IDS } from '@lukso/lsp-smart-contracts';
 
-const tokenType = await myAsset.getData('LSP4TokenType');
-console.log(tokenType);
-// 0 = Token
-// 1 = NFT
-// 2 = Collection
+const erc725js = new ERC725(
+  lsp4Schema,
+  '0x0514A829C832639Afcc02D257154A9DaAD8fa21B', // address of the digital asset contract
+  'https://4201.rpc.thirdweb.com',
+  {
+    ipfsGateway: 'https://api.universalprofile.cloud/ipfs',
+  },
+);
 ```
 
-<!-- prettier-ignore-end -->
-
-At this point, you should be able to identify if the digital asset is a:
+We can fetch the digital asset data in three ways:
 
-- LSP7 - Token
-- LSP7 - NFT
-- LSP8 - NFT
-- LSP8 - Collection
-
-## Fetch the Asset Metadata
+1. using [`getData()`](../../tools/erc725js/methods#getdata) without parameters. This will fetch the value of all data keys at once.
+2. using `getData("DataKeyName")` to fetch the value of a specific data key. For instance `getData("LSP4Metadata")`.
+3. using [`fetchData('LSP4Metadata')`](../../tools/erc725js/methods.md#fetchdata). This will decode the `VerifiableURI` to extract the link of the JSON metadata file and fetch its content from IPFS (or any other storage service).
```

Comment:
"using fetchData('LSP4Metadata'), which decodes the VerifiableURI to extract the JSON metadata file link and fetch its content from IPFS (or another storage service)."